### DISCUSSION TITLE
Installing Qt on centOS 7

### DISCUSSION THREAD

**Discussion by rubikstriangle** [AcademySoftwareFoundation/OpenRV#468] [Build Issues]
I'm trying to create a virtual box centOS7 builder machine.  It seems like the latest qt installer wants GLIBC_2.27 and the latest available for centOS 7 is GLIBC_2.25.  I can't see a way of downloading an older compatible installer.  Any ideas?    

This is what I get when I try to install:
./qt-online-installer-linux-x64-4.8.0.run 
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libstdc++.so.6: version `CXXABI_1.3.11' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)
./qt-online-installer-linux-x64-4.8.0.run: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by ./qt-online-installer-linux-x64-4.8.0.run)

**Comment by cedrik-fuoco-adsk** [comment]
You could use an older version of the installer from here :
https://download.qt.io/archive/online_installers/

I think version around 4.2 should work, but you'll have to try them manually. You might have to add new repository to get newer Qt version if you take an older unified installer, see here : https://stackoverflow.com/a/76446351
